\subsection*{Cloning and subcloning of the fly T-type Ca\textsuperscript{2+} channel (DmCa\textsubscript{v}3)}

Full length cDNA clone of DmCa\textsubscript{v}3 (CG15899) was generated by PCR amplification and subcloning.
Total RNA were extracted from adult heads using Trizol reagents (Invitrogen) and reverse transcribed into cDNA using RevertAid First Strand cDNA Synthesis Kit (Fermentas).
cDNA library was used as template for PCR amplification.
Primer sets were designed to obtain six adjacent DNA fragments (clone 1 -- clone 6) that cover full-length DmCa\textsubscript{v}3 CDS.
Primer design was based on Flybase gene annotation.
Hind III and Xba I were inserted at 5' of clone 1 and 3' of clone 6, respectively.
Primer sets used : clone 1 (5'-CGAGATAAGCTTAAAATGCTGCCACAGCCA-3' and 5'-GCATCAGACTACATCGCTGTC-3'), clone 2 (5'-CTGGACACGCTGCCCATGCTG-3' and 5'-TTCCAGCTCCTCCACTTGCAC-3'), clone 3 (5'-CAACGGTGGCTCCAACAGTCG-3' and 5'-CCACTGGCGGAAGCTCATGCC-3'), clone 4 (5'-GCCACGCCTCTCCAAGATCCG-3' and 5'-GACGATAAGAGCGTTTGCACG-3'), clone 5 (5'-TCTGAAACTAGTCGTGCAAAC-3' and 5'-TGGAAGTACTGGACGGTCTGC-3') and clone 6 (5'-AATCCCAGCCTGACCAGCTCG-3' and 5'-TCTAGATTAGTCCATGGAGGATTGGGGTGA-3').
Amplified PCR fragments were sequenced and assembled into pBlueScript II KS (+) vector by restriction enzyme digestion sequentially.
Among various isoform-specific fragments of clone 2 and clone 3, RC-specific fragments were used.
Frequent A to G RNA editing sites were found in clone 3 and 5.
One RNA editing site in clone 3(5'-AGTTCAGAGC-3') was reversed to A by site-directed mutagenesis based on genomic sequence.
Due to several RNA editing sites in clone 5, genomic DNA was used as template for clone 5 instead of cDNA.

Assembled full length cDNA was cut out of the vector by digestion of HindIII and XbaI and then subcloned into pcDNA3-HE3 where 5' untranslated region of \emph{Xenopus laevis} $beta$ globin was included for better expression in \emph{Xenopus} oocytes.